tests/unit/plugin/test_plugins.py

```
@@ -0,0 +1,1256 @@
+    def test_validate_generic_exception(self) -> None:
+        """validate() handles generic exceptions during spec lookup."""
+        entry = make_entry(class_path="test.module:Class")
+        with patch("importlib.util.find_spec", side_effect=ValueError("Test error")):
+            valid, error = entry.validate()
+            assert valid is False
+            assert "Error checking module" in error
+
+    def test_validate_with_check_class_valid(self) -> None:
+        """validate(check_class=True) validates class existence."""
+        entry = make_entry()
+        valid, error = entry.validate(check_class=True)
```

Ruff (0.14.14)

[warning] Unpacked variable `error` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)